app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java

```diff
@@ -139,11 +142,10 @@ public Fragment getItem(int position) {
                     return LemonCellExperiment.newInstance();
                 if (experimentTitle.equals(context.getString(R.string.capacitor_discharge)))
                     return CapacitorDischargeExperiment.newInstance();
-                if (experimentTitle.equals(context.getResources().getString(R.string.full_wave_rectifier)))
```

Comment:
you have removed case for full wave rectifier. Please add it again and replace hard-coded strings with string resources in both ```case 0``` and ```case 1```